app/assets/stylesheets/fortitude/api/tools/_mixins.scss

```diff
@@ -67,12 +68,44 @@
   td {
     border-style: unquote($border-style);
     border-width: $border-width;
-    line-height: $fortitude-base-line-height + $border-width;
-    padding: fortitude-component-padding($component-height, $border-width);
+    padding: fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height + 0.2rem, $border-width);
+  }
+
+  tr:last-child {
+    td,
+    th {
+      padding: fortitude-component-padding($component-height, $border-width);
+    }
   }
 
-  tbody tr:last-child td {
-    line-height: $fortitude-base-line-height;
+  @if (fortitude-contains($children, "thead") and fortitude-contains($children, "tbody")) {
+    // if there is a thead and tbody we have to compensate
+    thead + tbody tr:last-child {
+      td,
+      th {
+        padding: fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height + 0.2rem, $border-width);
+      }
+    }
+  }
+
+  @if (fortitude-contains($children, "tbody") and fortitude-contains($children, "tfoot")) {
+    // if there is a tbody and tfoot we have to compensate
+    tbody + tfoot tr:last-child {
+      td,
+      th {
+        padding: fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height, $border-width) fortitude-component-padding($component-height + 0.2rem, $border-width);
```

Comment:
This is 

![ktn-black-magic-sm](https://cloud.githubusercontent.com/assets/162151/7212308/8fdbda0c-e519-11e4-9c15-9b4702dea02b.jpg)